src/main/java/edu/ucsb/cs156/gauchoride/entities/Ride.java

```diff
@@ -18,13 +21,22 @@
 @NoArgsConstructor
 @Builder
 @Entity(name = "ride")
+@Table(name= "ride")
 public class Ride {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
     private String day;
-    private String studentName; 
-    private String driverName; 
+
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User rider; 
+
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User driver; 
+
+
     private String course;
     private LocalTime timeStart;

```

Comment:
add @JsonFormat(pattern = "HH:mm", shape = JsonFormat.Shape.STRING), because swagger UI's local time has an object shape and your backend doesn't how to serialize and deserialize that. so I believe you need this line to specify the rule. For both localTime fields. https://www.baeldung.com/jackson-jsonformat#:~:text=%40JsonFormat%20is%20a%20Jackson%20annotation,according%20to%20a%20SimpleDateFormat%20format.